composer.json

```diff
@@ -3,13 +3,20 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
         "wp-coding-standards/wpcs": "^2.1",
         "phpcompatibility/phpcompatibility-wp": "^2.1.0",
-        "automattic/vipwpcs": "^2.0"
+        "automattic/vipwpcs": "^2.0",
+        "phpunit/phpunit": "^5.7",
+        "brain/monkey": "^2.5"
```

Comment:
Why 2.5? As in `wordpress-seo` it is 2.4?

Reply:
Yep I noticed that. In `wpseo-news` is 2.5 though so I just used the most recent version we're using.